src/etc/inc/interfaces.inc

```diff
@@ -905,7 +909,7 @@ function interfaces_configure($verbose = false)
     $track6_list = array();
 
     /* This is needed to speedup interfaces on bootup. */
-    $reload = !file_exists('/var/run/booting');
+    $reload = file_exists('/var/run/booting') ? INTERFACE_BOOTING : INTERFACE_RELOAD_ALL;
```

Comment:
is that really necessary?

Reply:
https://github.com/kekek2/core/blob/c4b02532caf071a05bff96312041511906837920/src/etc/inc/interfaces.inc#L957

In this line uses variable $reload.